### PR TITLE
doc: Version update for NCS 1.9.0

### PR DESCRIPTION
**Pull request by mia-ko**
Changed main branch heading to nRF Connect SDK v1.9.0
NCSDK-13405

Signed-off-by: Mia Koen <mia.koen@nordicsemi.no>

### CODE REVIEW

**Review by rugeGerritsen — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

File: zboss/CHANGELOG.rst [hunk 1]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
 
-Main branch
-***********
+nRF Connect SDK v1.9.0
+**********************
 
-All the notable changes included in the |NCS| main branch are documented in this section.
+All the notable changes included in the |NCS| v1.9.0 release are documented in this section.
```
Comment (tomchy) [inline zboss/CHANGELOG.rst:15]: Please update the sentence in line 30th (possibly 57th) too:
```
For detailed information, see `ZBOSS stack release notes`_ for the main branch.
```
To:
```
For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.9.0 release branch.
```

**Review by grochu — APPROVED** [review]
Looks good.

File: zboss/CHANGELOG.rst (hunk 1 above)
Comment (mia-ko) [reply zboss/CHANGELOG.rst:15]: Changed:
- line 30 to For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.9.0.
- line 57 to For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.8.0.
Aligning with how it was done for NCS release v1.7.0

**Review by greg-fer — COMMENTED** [review]
@tomchy , @FilipZajdel , didn't you want to add two limitations to the ZBOSS changelog ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/zboss/CHANGELOG.html#limitations ), as per this list: https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+1.9.0

Could you comment here if this is the case?

**Comment by FilipZajdel** [comment]
> @tomchy , @FilipZajdel , didn't you want to add two limitations to the ZBOSS changelog ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/zboss/CHANGELOG.html#limitations ), as per this list: https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+1.9.0
> 
> Could you comment here if this is the case?

I think it should be added. Do you agree @tomchy ?

**Review by KAGA164 — APPROVED** [review]

**Comment by mia-ko** [comment]
> @tomchy , @FilipZajdel , didn't you want to add two limitations to the ZBOSS changelog ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/zboss/CHANGELOG.html#limitations ), as per this list: https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+1.9.0
> 
> Could you comment here if this is the case?

Talked to @FilipZajdel and @tomchy. These are considered more known issues than limitations, and they are addressed in the known issues PR: https://github.com/nrfconnect/sdk-nrf/pull/6611#issuecomment-1033812863

**Comment by greg-fer** [comment]
What about changes to nRF 802.15.4 Radio Driver? If no changes are needed, we need to add a line stating "No changes in this release.", as we did in the past (https://github.com/nrfconnect/sdk-nrfxlib/pull/595/files). Same goes for other changelog where no changes were made in this release. 

For radio driver, please check with @czeslawmakarski ...

**Comment by umapraseeda** [comment]
Shouldn't we update the changelog heading in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_dm/doc/CHANGELOG.html#main-branch?

**Comment by mia-ko** [comment]
> Shouldn't we update the changelog heading in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_dm/doc/CHANGELOG.html#main-branch?

Yes, I ran a search and that one was the only one that slipped. Fixed now :)

**Review by umapraseeda — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**